.dockerignore

```diff
@@ -0,0 +1,4 @@
+.git
+.DS_Store
+.env
+node_modules
```

Comment:
in some files you have new line and in some not
what about unifing this to always adding newline?